src/test/java/io/specto/hoverfly/junit/core/HoverflyConfigTest.java

```diff
@@ -10,11 +11,12 @@
 
     @Test
     public void shouldHaveDefaultSettings() throws Exception {
-        HoverflyConfig configs = HoverflyConfig.configs();
+
+        HoverflyConfiguration configs = configs().build();
 
         assertThat(configs.getHost()).isEqualTo("localhost");
-        assertThat(configs.getAdminPort()).isEqualTo(0);
-        assertThat(configs.getProxyPort()).isEqualTo(0);
+        assertThat(configs.getAdminPort()).isGreaterThan(0);
```

Comment:
Why has this changed? I think local should always be on a random port so there are no clashes